### PR TITLE
Add message type reply

### PR DESCRIPTION
**Pull request by neithernut**
This PR introduces the `ws_reply` types.
Targets #241.

### CODE REVIEW

**Comment by neithernut** [comment]
This should now be ready for merge.

File: src/objects/message/error_reply.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+/**
+ * Variable which holds the type information about the ws_error_reply type
+ */
+extern ws_object_type_id WS_OBJECT_TYPE_ID_ERROR_REPLY;
+
+/**
+ * Create a new error reply message
+ *
+ * This function creates a new error reply message.
+ */
```
Comment (matthiasbeyer) [inline src/objects/message/error_reply.h:62]: Return documentation missing.

**Comment by matthiasbeyer** [comment]
Good besides the one annotation.

**Comment by neithernut** [comment]
Fixed.

**Comment by matthiasbeyer** [comment]
ACK.

**Pull request merged**